crates/polars-core/src/chunked_array/logical/categorical/builder.rs

```diff
@@ -12,6 +12,12 @@ use crate::hashing::_HASHMAP_INIT_SIZE;
 use crate::prelude::*;
 use crate::{using_string_cache, StringCache, POOL};
 
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
Can you implement `Default` for this and set `Physical` to default?

There is syntactic sugar for this.